### DISCUSSION TITLE
[Projects Beta] Ability to clone/copy/duplicate projects or project templates?

### DISCUSSION THREAD

**Discussion by BenHenning** [community/community#21089] [Other Feature Feedback, Questions, & Ideas]
Hi,

I surprisingly couldn't find any other questions for this, but is it possible to clone a beta project board today or create one from a template? I'm finding that my team is trending toward setting up ~3 different kinds of project boards that we're likely going to repeat using for different sub-teams and projects that are being managed. Having an ability to create new projects based on a template or cloning a previous board would be extremely helpful in achieving this.

Are there any such plans in the works? Or, is this actually possible and I just missed the option?

Thanks!

**Comment by rwinchester** [comment]
I have the same issue. Was amazed to find that there seems to be no way to set up a Project Template that can be re-used as a starting point for new projects. So every time I make a new project (frequent!) I have to re-build what I want. Sad face.

**Comment by AntoineC4Ci** [comment]
Hello there,

I am also pretty surpised that project beta still not incude such a feature.
Is there any news about it ?

Regards

**Comment by terry-woodruff-hlpartnership** [comment]
Yes, we need to be able to copy a project including it's workflows too. 
Anywhere you can upvote this?

**Comment by jeremy-farrance** [comment]
We definitely need this. Almost all our projects start life with a big list of 80+ items (grouped by Categories). Setting it up each time is very time consuming.
